src/VirtualClient/VirtualClient.Main/profiles/GET-STARTED-REDIS.json

```diff
@@ -55,22 +55,10 @@
             }
         },
         {
-            "Type": "WgetPackageInstallation",
```

Comment:
Have you tested the changes in ubuntu?

Reply:
There seems to be some issue with Redis in ubuntu even without these changes, but the changes were running fine on AzLinux.